src/page-module/files/__name@dasherize__/__name@dasherize__.component.spec.ts

```diff
@@ -6,7 +6,8 @@ import { Store, StoreModule } from '@ngrx/store';
 import { TestBed } from '@angular/core/testing';<% } %>
 import { render, RenderResult } from '@testing-library/angular';<% if (isNgxTranslateInstalled) {%>
 import { configuration } from '@configurations';
-import { TranslateTestingModule } from 'ngx-translate-testing';<% } %>
+import { TranslateTestingModule } from 'ngx-translate-testing';<% } %><% if (facade) {%>
+import { <%= classify(section + ' ' + ((hasParent) ? (parent + ' ') : '') + name) %>PageFacade } from './<%= dasherize(name) %>.facade';<% } %>
```

Comment:
@Goodmain same as above, this `classify` call can be shortened.

Reply:
@dmitryusenko Fixed